### DISCUSSION TITLE
How to view blob information on the frontend

### DISCUSSION THREAD

**Discussion by qymlyl** [blockscout/blockscout#10149] [Q&A]
<img width="1184" alt="image" src="https://github.com/blockscout/blockscout/assets/61776278/6acb3c55-4619-45db-b7f9-003d963b5b69">
<img width="1000" alt="image" src="https://github.com/blockscout/blockscout/assets/61776278/fe39d48b-f6a0-48bb-b289-13f33602f5a6">

**Comment by sl1depengwyn** [comment]
Blobs are not included in the UI yet, however you can get this data via APIv2
